source/docs/advanced/production-ready/metrics.md

```diff
@@ -0,0 +1,19 @@
+---
+id: metrics
+title: Server's metrics
+---
+
+# Expose metrics for prometheus
+
+Front-Commerce provides a `/metrics` endpoint with data about the node process and response times.
+
+## Configure it
+
+You need two steps to enable this endpoint
+
+- `npm install express-prom-bundle prom-client`
+- define the environment variable `FRONT_COMMERCE_CLOUD_METRICS_KEY` with an basic authentication token (base64 encoded)
```

Comment:
Can you maybe link or and an example for the `(base64 encoded)` part as it might not be clear, like are there any specification needed in terms of character set, new line seperators etc etc.

Reply:
Changed for 
```
- define the environment variable `FRONT_COMMERCE_CLOUD_METRICS_KEY` with a basic authentication token (base64 encoded, e.g. `dXNlcjpwYXNzd29yZA==` for `user:password`)
```

is it better ?